tests/foreman/cli/test_activationkey.py

```diff
@@ -1368,4 +1361,12 @@ def test_positive_content_override(self):
                     u'organization-id': self.pub_key['org_id'],
                     u'value': override_value,
                 })
-                self.assertIn(result[0], 'Updated content override')
+                # Retrieve the product content enabled flag
+                result = ActivationKey.product_content({
+                    u'id': self.pub_key['key_id'],
+                    u'organization-id': self.pub_key['org_id'],
+                })
+                self.assertEqual(
```

Comment:
Can you document what the values are and why you're performing these changes to `result[3]`? It will be of tremendous help in the future while reviewing this.